### PR TITLE
Do not exclude the `status` attribute

### PR DESCRIPTION
**Pull request by julienduchesne**
I've encoutered a CRD with a `status` field but it's not being generated

### CODE REVIEW

File: pkg/model/modifiers.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ const (
 
 var propertiesWithoutModifiers = map[string]excludeType{
 	"apiVersion": excludeInRootOnly,
-	"status":     excludeEverywhere,
+	"status":     excludeInRootOnly,
```
Comment (jvrplmlmn) [inline pkg/model/modifiers.go:21]: For reviewers, this is the only change, the rest is just whitespace changes.

**Review by jvrplmlmn — COMMENTED** [review]
@julienduchesne I'm not so familiar with this codebase to give a proper review, is there something that we can run to verify that the behaviour is the expected one?

**Comment by Duologic** [comment]
Maybe we can make this an option we can enable per library? Doing it for all libraries might increase the library size quite a bit (especially for k8s-libsonnet).

**Comment by julienduchesne** [comment]
> Maybe we can make this an option we can enable per library? Doing it for all libraries might increase the library size quite a bit (especially for k8s-libsonnet).

I just searched for `withStatus` in `k8s-libsonnet` after generating it and found nothing (558 results across crossplane libs). Do you mean for the `status` attribute that's besides `spec`. This one is ignored because it's in the root of the objects

**Review by Duologic — APPROVED** [review]
Let's do this and see.

**Pull request merged**